js/components/closed-caption/captionOpacityTab.js

```diff
@@ -1,9 +1,9 @@
 var React = require('react'),
-    ClassNames = require('classnames'),
-    Utils = require('../utils'),
-    CONSTANTS = require('../../constants/constants'),
-    SelectionContainer = require('./selectionContainer'),
-    Slider = require('../slider');
+  ClassNames = require('classnames'),
```

Comment:
I think this is extensible as well with `indent: ["error", 2, { "VariableDeclarator": { "var": 2 } } ]`, should we add this change?

Reply:
Yep, I think this will help readability as well